bentoml/_internal/runner/runner_handle/remote.py

```diff
@@ -100,11 +99,9 @@ def strip_query_params(url: yarl.URL) -> str:
                 return str(url.with_query(None))
 
             jar = aiohttp.DummyCookieJar()
-            if timeout_sec is not None:
-                timeout = aiohttp.ClientTimeout(total=timeout_sec)
-            else:
-                DEFAULT_TIMEOUT = aiohttp.ClientTimeout(total=5 * 60)
-                timeout = DEFAULT_TIMEOUT
+            # uses only global timeout and not runner specific timeouts
+            timeout_from_config = BentoMLContainer.config.runners.timeout.get()
```

Comment:
Should we be using the per-runner timeout here?

Reply:
Initially decided we might skip this so that we can get the PR merged faster and take per-runner timeout later but I can look into this now. should not be that different.

Reply:
udpated